apps/OpenSpace/ext/launcher/include/sgctedit/sgctedit.h

```diff
@@ -0,0 +1,82 @@
+/*****************************************************************************************
+ *                                                                                       *
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2022                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#ifndef __OPENSPACE_UI_LAUNCHER___SGCTEDIT___H__
+#define __OPENSPACE_UI_LAUNCHER___SGCTEDIT___H__
+
+#include <QApplication>
+#include <QColor>
+#include <QDialog>
+#include <QLabel>
+#include <QLayout>
+#include <QLineEdit>
+#include <QMainWindow>
+#include <QComboBox>
+#include <QPushButton>
+#include <QScreen>
+#include <QTextBrowser>
+#include <QWidget>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <sgctedit/display.h>
+#include <sgctedit/filesupport.h>
+#include <sgctedit/monitorbox.h>
+#include <sgctedit/orientation.h>
+
+class SgctEdit final : public QDialog
+{
+Q_OBJECT
+public:
+    SgctEdit(QWidget* parent, std::vector<sgct::config::Window>& windowList,
+        sgct::config::Cluster& cluster, const QList<QScreen*> screenList);
```

Comment:
`screenList` can be declared as `const QList<QScreen*>&` instead.

Maybe it doesn't need to be passed in at all and the `systemMonitorConfiguration` can just ask the application directly?